scripts/post-install/post-install.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const Logger = require('../utils/logger');
+
+fs.readFile(`${process.env.INIT_CWD}/package.json`, (error, file) => {
+  if (error) {
+    throw Logger.error(error);
+  }
+
+  const { dependencies } = JSON.parse(file);
+
+  if (dependencies['terra-toolkit']) {
+    Logger.warn('+-------------------------------------------------------------------------------------------------------------+');
+    Logger.warn('|                                                   WARNING                                                   |');
+    Logger.warn('+-------------------------------------------------------------------------------------------------------------+');
+    Logger.warn('|                            terra-toolkit should be installed as a devDependency.                            |');
```

Comment:
`should` -> `must`

Reply:
https://github.com/cerner/terra-toolkit/pull/326/commits/bfb2643f266a8596dbfeeccdbf18d7c05969087b